pkgs/development/python-modules/aetcd/default.nix

```diff
@@ -28,7 +28,8 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace-fail "setuptools_scm==6.3.2" "setuptools_scm"
+      --replace-fail "setuptools_scm==6.3.2" "setuptools_scm"\
+      --replace-fail "protobuf>4,<5" "protobuf"
```

Comment:
This was already fixed with https://github.com/NixOS/nixpkgs/commit/dcd5f162a6754db89a4a2db44094687cf97dcfac (currently in `staging-next`).